### PR TITLE
MongoDB Dockerfile for ppc64le

### PR DESCRIPTION
**Pull request by ManirajDeivendran**
Dockerfile to create mongo client image version 4.4.

### CODE REVIEW

File: m/mongodb/Dockerfiles/rhel_8.3/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,135 @@
+FROM ppc64le/ubuntu:20.04
```
Comment (seth-priya) [inline m/mongodb/Dockerfiles/rhel_8.3/Dockerfile:1]: the base image here is ubuntu:20.04 as opposed to RHEL or UBI, which is fine for now but we might have to think of creating an image with UBI base in future. For now, please correct the folder structure. This file should reside in an ubuntu_20.04 folder under Dockerfiles.

File: m/mongodb/Dockerfiles/rhel_8.3/docker-entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,383 @@
+#!/bin/bash
+set -Eeuo pipefail
```
Comment (seth-priya) [inline m/mongodb/Dockerfiles/rhel_8.3/docker-entrypoint.sh:2]: General comment on this and the dockerfile - has this been adapted from a similar version already available for other architectures elsewhere? If so, we should have a clear attribution as a comment at the top of each file mentioning its origin

**Review by seth-priya — CHANGES_REQUESTED** [review]

File: m/mongodb/Dockerfiles/rhel_8.3/Dockerfile (hunk 1 above)
Comment (ManirajDeivendran) [reply m/mongodb/Dockerfiles/rhel_8.3/Dockerfile:1]: Understood. Resolved.

File: m/mongodb/Dockerfiles/rhel_8.3/docker-entrypoint.sh (hunk 2 above)
Comment (ManirajDeivendran) [reply m/mongodb/Dockerfiles/rhel_8.3/docker-entrypoint.sh:2]: Add comment with origin information.

**Comment by ManirajDeivendran** [comment]
Fixed review comments.

**Review by seth-priya — APPROVED** [review]
Changes look good, please squash the commits

**Comment by ManirajDeivendran** [comment]
Created new PR https://github.com/ppc64le/build-scripts/pull/1000.

**Pull request closed without merging**